openedx/core/djangoapps/programs/admin.py

```diff
@@ -0,0 +1,15 @@
+"""
+django admin pages for program support models
+"""
+from django.contrib import admin
+
+from config_models.admin import ConfigurationModelAdmin
+
+from openedx.core.djangoapps.programs.models import ProgramsApiConfig
+
+
+class ProgramsApiConfigAdmin(ConfigurationModelAdmin):  # pylint: disable=missing-docstring
```

Comment:
can we add search to this if we need to find people it will make our lives easier.